Pages/Index.cshtml

```diff
@@ -73,31 +73,34 @@
-    </div>
+<div class="container" style="padding: 35px">
+    <h3 class="text-center w-100 text-dark h3 mt-5 mb-2 ">Safest Neighborhoods In Cincinnati</h3>
+    <div class="row g-3">
+        <div class="col-4 ">
+            <img src="https://i0.wp.com/haushomemagazine.com/wp-content/uploads/2021/04/My-Post-6.jpg?w=1200&ssl=1" class="zoom-hover rounded" alt="Hyde Park, Cincinnati" />
+            <h4 class="h4 text-center">Hyde Park, Cincinnati</h4>
+        </div>
 
-    <div class="col-4">
-        <img src="https://abccincy.org/wp-content/uploads/2016/02/Sayler-Park-768x576.jpg" class="zoom-hover rounded" alt="image of place" />
-        <h4 class="h4 text-center mt-2">Sayler Park</h4>
-    </div>
-    <div class="col-4">
-        <img src="https://www.homesnacks.com/images/oh/cincinnati-oh-11.jpg" class="zoom-hover rounded" alt="image of place" />
-        <h4 class="h4 text-center mt-2">Mt. Washington</h4>
-    </div>
-    <div class="col-4">
-        <img src="https://mycincinnatirealtor.com/files/2019/12/Observatory-2-2-1024x680.jpg" class="zoom-hover rounded" alt="image of place" />
-        <h4 class="h4 text-center mt-2">Mt. Lookout</h4>
-    </div>
-    <div class="col-4">
-        <img src="https://th.bing.com/th/id/OIP.lShCGT7aAXVlnSxBC4tbXgHaFb?pid=ImgDet&rs=1" class="zoom-hover rounded" alt="image of place" />
-        <h4 class="h4 text-center mt-2">Kennedy Heights</h4>
+        <div class="col-4">
+            <img src="https://abccincy.org/wp-content/uploads/2016/02/Sayler-Park-768x576.jpg" class="zoom-hover rounded" alt="Sayler Park" />
+            <h4 class="h4 text-center mt-2">Sayler Park</h4>
+        </div>
+        <div class="col-4">
+            <img src="https://www.homesnacks.com/images/oh/cincinnati-oh-11.jpg" class="zoom-hover rounded" alt="Mt. Washington" />
+            <h4 class="h4 text-center mt-2">Mt. Washington</h4>
+        </div>
+        <div class="col-4">
+            <img src="https://mycincinnatirealtor.com/files/2019/12/Observatory-2-2-1024x680.jpg" class="zoom-hover rounded" alt="Mt. Lookout" />
+            <h4 class="h4 text-center mt-2">Mt. Lookout</h4>
+        </div>
+        <div class="col-4">
+            <img src="https://th.bing.com/th/id/OIP.lShCGT7aAXVlnSxBC4tbXgHaFb?pid=ImgDet&rs=1" class="zoom-hover rounded" alt="Kennedy Heights" />
+            <h4 class="h4 text-center mt-2">Kennedy Heights</h4>
+        </div>
+        <div class="col-4">
+            <img src="https://th.bing.com/th/id/OIP.V3k0IXS0gAzcSNiSGiSuDgHaE6?pid=ImgDet&rs=1" class="zoom-hover rounded" alt="University Of Cincinnati" />
+            <h4 class="h4 text-center mt-2">University Of Cincinnati</h4>
+        </div>
     </div>
-    <div class="col-4">
-        <img src="https://th.bing.com/th/id/OIP.V3k0IXS0gAzcSNiSGiSuDgHaE6?pid=ImgDet&rs=1" class="zoom-hover rounded" alt="image of place" />
-        <h4 class="h4 text-center mt-2">University Of Cincinnati</h4>
-    </div>
-</div>
+
+</div>
```

Comment:
What changed here?  Too many changed lines to determine how it reduces tech debt.  This is where a conversation here in the Files Changed section is handy.